gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/EmailServiceImpl.java

```diff
@@ -261,7 +261,7 @@ private Map<String, Object> prepareEmailParams(Domain domain, Application client
         }
 
         params.put("url", domainService.buildUrl(domain, redirectUri, queryParam));
-
+        params.put("gatewayHost", domainService.gatewayUrl().getHost().replace("://www.", "://"));
```

Comment:
**todo**: For a URL `http://www.gravitee.io/path`, `URI.getHost()` will provide  `www.gravitee.io`.
So if I understand the intension (provide a vairable in the email template engine to make the noreply adapted to the domain), maybe the following change is more appropriate.

```suggestion
        params.put("gatewayHost", domainService.gatewayUrl().getHost().replaceFirst("^www\\.", ""));
```